posts/saving-and-analyzing-trending-topics-on-twitter-using-aws-athena-lambda-and-cdk.md

```diff
@@ -0,0 +1,198 @@
+---
+authors:
+- Theo Lebrun
+tags:
+- Twitter
+- AWS
+- AWS Lambda
+- AWS Athena
+- AWS CDK
+date: 2020-08-03T14:50:55.000Z
+title: "Saving and analyzing trending topics on Twitter using AWS Athena, Lambda and CDK"
```

Comment:
"Saving and Analyzing Trending Topics on Twitter using AWS Athena, Lambda, and CDK"

[capitalized "Analyzing," "Trending," and "Topics]
[inserted comma after "Lambda"]